llvm/lib/Target/EraVM/AsmParser/EraVMAsmParser.cpp

```diff
@@ -328,6 +328,7 @@ static int parseExplicitCondition(StringRef Code) {
       .Case("ne", EraVMCC::COND_NE)
       .Case("ge", EraVMCC::COND_GE)
       .Case("le", EraVMCC::COND_LE)
+      .Case("of", EraVMCC::COND_LT)
```

Comment:
We don't plan to support this alias in asm.

Reply:
Removed.